core/templates/dev/head/css/oppia.css

```diff
@@ -4369,21 +4369,38 @@ div.oppia-issues-learner-action {
 }
 
 .oppia-issues-playthrough-modal-btn {
-  background: transparent;
-  border-radius: 0;
+  padding: 15px 25px;
```

Comment:
@SamriddhiMishra I took a quick pass and I've a few suggestions:
1. Add the style/properties in sorted order (with respective to the names).
2. As this is a UI related PR, please add a screenshot (before/after) for the relevant parts. (This will help codeowners to review your PR quickly.)